src/main/resources/application.yaml

```diff
@@ -44,6 +44,8 @@ storage:
   account_name: ${STORAGE_ACCOUNT_NAME:bulkscanning}
   key: ${STORAGE_KEY:dGVzdA==} #Base 64 encoded
   blob_lease_timeout: ${STORAGE_BLOB_LEASE_TIMEOUT:15}
+  signature_algorithm: ${STORAGE_BLOB_SIGNATURE_ALGORITHM:none}  # none or sha256withrsa
```

Comment:
flag should be to unable disable algorithm and not to give a choice for algorithm

Reply:
Please do read the specs before commenting.

Reply:
I don't agree with this one. Please can you confirm with Neil